standards/feature-definition-of-done.md

```diff
@@ -0,0 +1,10 @@
+# Feature Definition of Done
+
+A feature is done when the following criteria are met.
+
+1. The code compiles without errors
+2. The software built successfully in a CI process
```

Comment:
I think 2. can be written more concise. 
The usage of built and build in relation to software and the CI process can confuse here.
Maybe "The software has been build successfully in a CI process"?